src/scalib_ext/scalib/src/sasca/fg_parser.rs

```diff
@@ -21,11 +21,12 @@ impl NVar {
 pub(super) enum Expr {
     Not(Var),
     Lookup { var: Var, table: String },
-    Add(Vec<Var>),
+    Add(Vec<NVar>),
     Mul(Vec<Var>),
     Xor(Vec<Var>),
     And(Vec<NVar>),
     Or(Vec<NVar>),
+    Sub(Vec<NVar>),
```

Comment:
The meaning of `Sub([a, b, c])` is not really clear (is it `a-b-c`, or `a-(b-c)`, or ...?). Further, the implementation of the parser doesn't seem to support `a-b+c`.